css/admin.css

```
@@ -2279,6 +2363,329 @@
+      padding: 12px;
+      border-radius: 14px;
+      border: 1px solid rgba(0, 0, 0, 0.08);
+      background: var(--onboarding-theme-bg);
+      color: var(--onboarding-theme-ink);
+    }
+
+    .admin-onboarding-theme-preview::before,
+    .admin-onboarding-theme-preview::after {
+      content: "";
+      border-radius: 999px;
+      background: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)